test/common/upstream/BUILD

```diff
@@ -43,6 +43,8 @@ envoy_cc_test(
         "//source/common/stats:stats_lib",
         "//source/common/upstream:cluster_factory_lib",
         "//source/common/upstream:cluster_manager_lib",
+        "//source/extensions/filters/network/echo",
```

Comment:
Is this change still used?

Reply:
Good catch, it's not. Would be nice for bazel to warn (like it does in blaze).